### PR TITLE
Update documentation for changing DNS for *.service.gov.uk

### PR DESCRIPTION
**Pull request by brucebolt**
This adds all the steps needed and assumes the person following the instructions has never run this before.

It also gives examples of what to check in the `plan` output.

### CODE REVIEW

File: source/manual/dns.html.md [hunk 1]
```diff
@@ -45,39 +45,106 @@ The deployment process is complex, so it's much easier to [use the automation pr
...
+
+1. Install gcloud:
+
+    ```
+    brew install --cask google-cloud-sdk
+    source "$HOMEBREW_PREFIX/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/path.zsh.inc"
+    ```
 
-This will trigger a build in Jenkins. Check the build output to see what terraform has done / would do.
+1. Login to the gcloud CLI by running the followng, then logging into your GDS Google account in the browser window that will open:
```
Comment (leenagupte) [inline source/manual/dns.html.md:98]: Typo: "following"

File: source/manual/dns.html.md [hunk 2]
```diff
@@ -45,39 +45,106 @@ The deployment process is complex, so it's much easier to [use the automation pr
...
+    export GITHUB_USERNAME=<your_github_username>
+    ```
 
-OPTIONS:
-...
-```
+1. Check you can access AWS (in production as a poweruser) using the following, if not, follow the steps in the [Get Started documentation](/manual/get-started.html#8-get-aws-access).
+
+    ```
+    gds aws govuk-production-poweruser aws s3 ls
```
Comment (leenagupte) [inline source/manual/dns.html.md:60]: If this command is successfully, should I be able to see a list of jobs?
For example:

```
2020-01-27 11:52:59 aptly-sync
2022-12-20 08:15:29 aws-athena-query-results-172025368201-eu-west-1
2022-07-15 12:00:27 datagovuk-production-ckan-organogram
2020-06-24 03:03:40 datagovuk-production-ckan-static-data
2020-06-24 04:07:20 dns-state-bucket-production
2020-06-24 08:14:03 gds-public-readable-tarballs
2023-01-04 15:23:43 govuk-analytics-logs-production
.
.
.
```

File: source/manual/dns.html.md [hunk 3]
```diff
@@ -45,39 +45,106 @@ The deployment process is complex, so it's much easier to [use the automation pr
...
 
-### Deploying a change
+    ```
+    gcloud auth login
+    ```
+
+1. Run a plan in GCP:
+
+    ```
+    gds govuk dns --provider gcp --zone service.gov.uk --action plan --role govuk-production-poweruser
```
Comment (leenagupte) [inline source/manual/dns.html.md:107]: Perhaps a line to say that running a plan is the equivalent of testing it, and no actual changes are made until the "apply" stage?

**Review by leenagupte — COMMENTED** [review]
The instructions are great and really clear, thanks!

I tried out all of the commands except for the `apply` ones, and they all work as expected.

I've added one suggestion inline.

File: source/manual/dns.html.md (hunk 1 above)
Comment (brucebolt) [reply source/manual/dns.html.md:98]: Oops.  Will fix.

File: source/manual/dns.html.md (hunk 2 above)
Comment (brucebolt) [reply source/manual/dns.html.md:60]: Yes, a list of S3 buckets.  Will add a sentence explaining what you should see.

File: source/manual/dns.html.md (hunk 3 above)
Comment (brucebolt) [reply source/manual/dns.html.md:107]: Good idea.  I'll add this.

**Review by leenagupte — APPROVED** [review]
A very easy read, thanks! 🎉

**Pull request merged**